### PR TITLE
Update draft-boucadair-nmop-rfc3535-20years-later.md - External depen…

### PR DESCRIPTION
**Pull request by luismcontreras**
…dencies

Proposed content in External dependecies

### CODE REVIEW

File: draft-boucadair-nmop-rfc3535-20years-later.md [hunk 1]
```diff
@@ -233,9 +233,11 @@ TBC.
 
 ## Implications of External Dependency (Luis)
 
-TSN/detenet
+Current networks are abandoning the silo approach from the past towards an increasing convergence. On one hand, there are trends towards a tighter interaction and integration of different technologies previously considered as totally separated from an operational perspective. Examples of that are the IP and Optical integration (e.g., the introduction of coloured interfaces on routers), or the extension of deterministic-behavior features to layer 3 networks. This kind of convergence in most cases creates dependecies on the traditional network management features, which require to incoprorate or integrate functionality from other technological domains.
```
Comment (boucadair) [inline draft-boucadair-nmop-rfc3535-20years-later.md:236]:
```suggestion
Networks are being updated to abandon the silo approach from the past towards an increasing convergence. Specifically, there are trends towards a tighter interaction and integration of different technologies previously considered as totally separated from an operational perspective. Examples of that trends are the IP and Optical integration (e.g., the introduction of colored interfaces on routers), or the extension of deterministic-behavior features to Layer 3 networks. This kind of convergence in most cases creates dependencies on the conventional network management features, which require to incorporate or integrate functionality from other technological domains.
```

File: draft-boucadair-nmop-rfc3535-20years-later.md [hunk 2]
```diff
@@ -233,9 +233,11 @@ TBC.
...
 
-TBC.
+Furthermore, such convergence is also reflected on the need of interacting and interworking with distinct network parts participating of the end-to-end service delivery. Mobile access, fixed access, data center, enterprise, radio functional split (i.e., fronthaul and midhaul), neutral exchanges, intensive data networks (e.g., scientific academic networks), content distribution, etc, represent network parts constituent of end-to-end services that can impose dependencias of the management of an intermmediate network.
```
Comment (boucadair) [inline draft-boucadair-nmop-rfc3535-20years-later.md:238]:
```suggestion
Furthermore, such convergence is also reflected on the need of interacting and interworking with distinct network parts participating in the end-to-end service delivery. Mobile access, fixed access, data center, enterprise, radio functional split (i.e., fronthaul and midhaul), neutral exchanges, intensive data networks (e.g., scientific academic networks), content distribution, etc., represent network parts constituent of end-to-end services that can impose dependencies of the management of an intermediate network.
```

File: draft-boucadair-nmop-rfc3535-20years-later.md [hunk 3]
```diff
@@ -233,9 +233,11 @@ TBC.
...
+
+That convergence shown the last years also implies the need of an up-to-date refresh of management capabilities and tooling of the traditional networks.
```
Comment (boucadair) [inline draft-boucadair-nmop-rfc3535-20years-later.md:240]:
```suggestion
That convergence shown the last years also implies the need of an up-to-date refresh of management capabilities and tooling of the conventional networks. Also, it highlights the need to easily map the data models that are used to manage each specific segment.
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks, Luis. Please find some change proposals.

**Comment by luismcontreras** [comment]
Suggested changes are fine for me, thanks!

**Pull request merged**